roles/openshift_hosted/templates/v3.8/glusterfs-registry-endpoints.yml.j2

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: {{ openshift.hosted.registry.storage.glusterfs.endpoints }}
```

Comment:
```
TASK [openshift_hosted : Generate GlusterFS registry endpoints] ***************************************************************************************************************************************************
task path: /home/ec2-user/openshift-ansible/roles/openshift_hosted/tasks/storage/glusterfs_endpoints.yml:2
Wednesday 06 December 2017  18:44:16 +0000 (0:00:00.055)       0:01:06.218 **** 
fatal: [ec2-34-203-220-245.compute-1.amazonaws.com]: FAILED! => {
    "changed": false, 
    "failed": true, 
    "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'hosted'"
}
```

Reply:
this is environment variable openshift_hosted_registry_storage_glusterfs_endpoints which can be overwritten by user which comes from: https://github.com/openshift/openshift-ansible/blob/03f5d8249478ca5e62eb3d5db7aa1b7a43de96eb/roles/openshift_facts/defaults/main.yml
and because it's being set in defaults it never being translated to fact `syntax stuff.stuff.morestuff`  I just reused this value.

Reply:
There has been some other reorganization around this `.hosted` fact setting and it may just not be there any more. At least I got this error when I tried yesterday. You may want to have a look at recent git logs...

Reply:
Pretty sure I was calling the openshift-hosted playbook directly, not via the byo/config.yml, in case that matters.

Reply:
my approach would be to fix it to use _value_ as it should sort this one for now, and leave [yak shaving](http://www.dailymotion.com/video/x2gp98t)  for later :) If @jarrpa agree I can pick this as a separate ticket/issue.

Reply:
@mjudeikis What is _value_ in this case? If you mean using the `openshift_hosted_` format, then please, go ahead! Please also update the 3.7 and 3.6 templates in the master branch.

Reply:
Also CC me on that PR, please. :)

Reply:
you dont need to change 3.7. They already using `_hosted_` format https://github.com/openshift/openshift-ansible/pull/6385 .